cmd/maya-apiserver/spc-watcher/lease.go

```diff
@@ -49,35 +99,76 @@ type spcLease struct {
 	leaseKey string
 	// oecs is the openebs clientset
 	oecs clientset.Interface
+	// kubeclientset is a standard kubernetes clientset
+	kubeclientset kubernetes.Interface
 }
 
 func (sl *spcLease) GetLease() (string, error) {
```

Comment:
LeaseHolder() vs GetLease()
Can the return types be named?
e.g. `func (s *splLease) LeaseHolder() (holder string, err error)`